ci-operator/step-registry/telcov10n/functional/cnf-network/cnf-config/telcov10n-functional-cnf-network-cnf-config-ref.yaml

```diff
@@ -0,0 +1,14 @@
+ref:
+  as: telcov10n-functional-cnf-network-cnf-config
+  from_image:
+    namespace: telcov10n-ci
+    name: eco-ci-cd
+    tag: eco-ci-cd
+  commands: telcov10n-functional-cnf-network-cnf-config-commands.sh
+  timeout: 4h
+  resources:
+    requests:
+      cpu: 100m
+  env:
+  - name: CLUSTER_NAME 
+    default: "kni-qe-92"
```

Comment:
I think we should remove the default here (making the job fail if it's not provided), so that the user is forced to provide that env var, I assume we will have more clusters soon.
WDYT?